easypermissions/maven.gradle

```diff
@@ -39,6 +39,18 @@ task sourcesJar(type: Jar) {
     classifier = 'sources'
 }
 
+task javadoc(type: Javadoc) {
```

Comment:
I think we need to add [`failOnError = false`](https://docs.gradle.org/3.3/dsl/org.gradle.api.tasks.javadoc.Javadoc.html#org.gradle.api.tasks.javadoc.Javadoc:failOnError) for the build to work.